default/serializers.py

```
@@ -0,0 +1,68 @@
+from rest_framework import serializers
+from django.contrib.auth.models import User
+from .models import Post, Comment
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['id', 'username', 'email']
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)